src/plugins/data/common/search/aggs/buckets/range.ts

```diff
@@ -36,12 +36,14 @@ export interface RangeBucketAggDependencies {
   getFieldFormatsStart: AggTypesDependencies['getFieldFormatsStart'];
 }
 
+export interface RangeType {
```

Comment:
I think this change is triggering apparch review, you should be able to remove it for now as we moved out the custom labels

Reply:
I thought about that, but I need to reuse the `RangeType` interface as well without duplicating the type declaration.

Reply:
I see, missed that usage, thanks

Reply:
The basic types for ranges are already duplicated in many places (including `expressions` plugin which also has a `Range` type with a corresponding expression type).

While it makes sense to me to have something that can be shared, I don't know if this specific instance from the aggs service is the right place for it to live / the right one to use, and I'd rather not add something else to the data plugin's contract without having a plan for that.

Since you are already extending it anyway, I wonder if you might consider using the type from the `expressions` plugin as a starting point instead? As that wouldn't require changing the `data` plugin contracts.

Reply:
I can use that one.
It is a starting point but I have to customise it locally because of unused properties, like `type`